.github/workflows/stress.yml

```diff
@@ -2,6 +2,20 @@ name: Stress
 
 on:
   workflow_dispatch:
+    inputs:
+      logprofile:
+        description: 'Log profile to use'
+        required: false
+        default: "None"
+        type: choice
+        options:
+          - "None"
+          - SpinQuic.Light
```

Comment:
Wouldn't this one make more sense as a default?
Or if it isn't enough (look like it might contain very little), maybe we should consider adding what makes sense to it - no point in having a SpinQuic profile that is not enough to diagnose issues.